README.md

```
@@ -1,378 +1,209 @@
+| `file_tags` | Key-value tags |
+| `uploads` | Multipart upload tracking |
+| `semantic_tasks` | Background work queue (embedding, image extraction) |
 
-## Building
+## Mount options
 
-```bash
-mkdir -p bin
-go build -o bin/drive9 ./cmd/drive9
-go build -o bin/drive9-server ./cmd/drive9-server
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)